crates/net/rpc-builder/src/lib.rs

```diff
@@ -182,13 +185,44 @@ impl RpcServerBuilder {
     /// This consumes the builder and returns a server.
     ///
     /// Note: The server ist not started and does nothing unless polled, See also
-    pub async fn build(self) -> Result<RpcServer, RpcError> {
-        todo!()
+    pub async fn build(self) -> Result<Vec<RpcServer>, RpcError> {
+        let mut servers = Vec::new();
+        let socket_addr = self.http_ws_addr.unwrap();
+        let ips_endpoint = self.ipc_server_path.unwrap();
+
+        if let Some(builder) = self.http_ws_server_config {
+            let http_ws_server = builder.build(socket_addr).await?;
+            servers.push(RpcServer { server: RpcServerEnum::HttpWs(http_ws_server) })
```

Comment:
Let's have separate configs for HTTP and for WS, sometimes one may not want to bring up both, or may want to expose different ports for each one.